backend/package.json

```
@@ -4,28 +4,35 @@
   "description": "",
   "main": "start_backend.js",
   "scripts": {
-    "test": "jest",
+    "test": "jest --forceExit",
     "dev": "node start_backend.js",
-    "lint": "./node_modules/.bin/eslint \"**/*.js\""
+    "lint": "./node_modules/.bin/eslint \"**/*.js\"",
```

```suggestion
    "lint": "eslint \"**/*.js\"",
```
`yarn run x` or `npm run x` will *always* look into that folder to resolve the binary.